client/lib/feed-stream-store/index.js

```diff
@@ -21,15 +20,15 @@ function feedKeyMaker( post ) {
 	return {
 		feedId: post.feed_ID,
 		postId: post.ID,
-		localMoment: moment( post.date ),
+		date: new Date( post.date ),
```

Comment:
I believe the original reason was because `moment` was thought to be relatively expensive so we didn't want to create it on the individual react components.

I believe that ReaderPostCard (or a child) depends on the moment right now